src/minerva_lib/blend.py

```diff
@@ -1,59 +1,58 @@
 import numpy as np
+import skimage.exposure
 
 
-def to_f32(img):
-    '''Scale the dynamic range to 0.0 - 1.0
+def composite_channel(a, image, color, range_min, range_max, out=None):
+    ''' Composite given image _a_ with rendered channel _image_
 
-    Arguments:
-    img: An integer image
-    '''
+    To update _a_ destructively, pass the same array to _a_ and _out_.
 
-    # No well-defined behavior for decimal values or values less than 0
-    try:
-        dtype_info = np.iinfo(img.dtype)
-        assert dtype_info.min is 0
-    except (ValueError, AssertionError):
-        raise ValueError('Scaling to 0,1 requires unsigned integers')
+    Args:
+        a: Numpy array to composite
+        image: Numpy array of channel to render and composite
+        color: Color as r, g, b float32 array within 0, 1
```

Comment:
Do these really need to be float32 or will any float type actually work? If any type will work, strike the `32`.